packages/next/build/webpack-config.ts

```diff
@@ -582,6 +582,22 @@ export default async function getBaseWebpackConfig(
                         // anchor. By inserting before and not after, we do not
                         // need to track the last inserted element.
                         parentNode.insertBefore(element, anchorElement)
+
```

Comment:
just a note - `webpack-config.ts` gets compiled for the current version of Node, but this function gets stringified and executed on the client. It seems like that'll result in `const` and potentially the arrow function being used as-is?

Reply:
https://github.com/zeit/next.js/pull/8937/commits/f936b961b7f595744a31fcbaa8512fdfc4f44860